### DISCUSSION TITLE
how to get Site ID?

### DISCUSSION THREAD

**Discussion by DavidKorpiewski** [Security-Onion-Solutions/securityonion#9795] [General]
Newb question, but one that eludes me and even ChatGPT.  How do you get the Site ID out of a standalone installation of SO?   I'm trying to run PCAP, but it won't run without a site ID.   When I use so-status and other recommended tools, none of these show me a site ID that I can find.  You would have thought that the web interface would have had one, but I can't find it for the life of me. 
Please help.
David

**Comment by InfosecGoon** [comment]
Do you mean the Sensor ID in the PCAP screen? That's the name of the standalone or forward node that you'd like to pull the PCAP files from - it's listed under ID in the Grid screen.

**Reply by DavidKorpiewski** [reply]
Thank you! yes this is what I needed.  The site Id is just "securityonion"
